stonesoup/platform/base.py

```diff
@@ -150,6 +158,29 @@ def orientation(self, value):
     def __getitem__(self, item):
         return self.movement_controller.__getitem__(item)
 
+    @property
+    def ground_truth_path(self) -> GroundTruthPath:
+        """ Produce a :class:`.GroundTruthPath` with the same `id` and `states` as the platform.
```

Comment:
This doc string doesn't show well in the rendered docs

Reply:
Is that better?/Good enough